examples/python-handler/handler/schema.py

```
@@ -31,4 +31,34 @@ def resolve_subsystem(self, info):
         _subsystem.refresh()
         return _subsystem
 
-schema = graphene.Schema(query=Query)
+
+class Subsystem(graphene.Mutation):
+    """
+    Creates mutation for Subsystem
+    """
+
+    class Arguments:
+        power_on = graphene.Boolean()
+
+    Output = Status
+
+    def mutate(self, info, power_on):
+        """
+        Handles request for subsystem powerOn mutation
+        """
+
+        status = Status(status=True, subsystem=_subsystem)
+        if power_on != None:
```

This should probably throw an error

---

Do you mean it should throw an error if `power_on` is `None`? That is actually a correct state, as it means the graphql mutation does not affect `power_on`.

---

Yup that is correct.